### ISSUE TITLE
Fix VoiceOver experience

### ISSUE THREAD

**Issue by Touchgram** [Touchgram/purrticles#25]
Voiceover is unusable at present because the main editing panels mostly use our custom control that mimics the Xcode control.

It combines a stepper with a central label that can be tapped to edit.

The VoiceOver experience for each of these lacks context.

A possible fix is to use a rotor to pick between increase, decrease and edit.

Anyone who wants to help can see the basic code in a related sample [DocUndoable](https://github.com/AndyDentFree/swiftgooey/tree/main/Docundoable). Whilst primarily about undo, that sample uses these controls.
